### ISSUE TITLE
Way to exclude a single grid?

### ISSUE THREAD

**Issue by niners52** [mage-eag/mage-enhanced-admin-grids#320]
Enhanced Admin Grids breaks the grid for https://amasty.com/magento-smart-review-reminder.html 
Is there a way to just exclude certain pages from using Enhanced Admin Grids?

**Comment by seansan** [comment]
please check config first

**Comment by niners52** [comment]
Could you be more specific? Which config? And what am I checking?

**Comment by mickymix** [comment]
On the backpanel, go to System > Configuration > Enhanced Admin Grids > Base Configuration
Here you'll find a list of the Global Exclusions under the General tab...

You might disable EAG on the grid you want from here...

I guess this is written somewhere in the documentation and/or the wiki...

**Comment by niners52** [comment]
That is exactly what I was looking for. I looked in the Wiki but didn't see anything. Do you know where the documentation is? I am trying to figure out the difference between Global Exclusions and Complementary Exceptions and what should the block type and rewriting class be? I tried to Google it but couldn't find anything.

**Comment by mickymix** [comment]
You are right, the wiki seems not-existant, but I have a memory of having seen it in the past... can't figure our where, at this point... probably it was published in the help files of some Magento hosting service such as BoostMyShop or similar...
And for the second part of your question, sorry, I don't know it either... :-D
